cpu/esp8266/Makefile.features

```diff
@@ -7,3 +7,4 @@ FEATURES_PROVIDED += periph_cpuid
 FEATURES_PROVIDED += periph_hwrng
 FEATURES_PROVIDED += periph_pm
 FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += ssp
```

Comment:
@gschorcht ssp doesn't work properly on esp8266, right? (@nmeum it was blacklisted before)

Reply:
It does seem to the pass on the CI, see:

* https://github.com/RIOT-OS/RIOT/pull/13175#issuecomment-577146087
* https://ci.riot-os.org/RIOT-OS/RIOT/13175/4f8f280e1f45301b82b57c84002eb4db97425636/output.html

Reply:
esp8266 recently got a toolchain update (https://github.com/RIOT-OS/riotdocker/pull/93), that might have fixed it.

Reply:
Hm, I hadn't tested it yet for the new platform implementation that uses a newer GCC version. The result of 
```
CONTINUE_ON_EXPECTED_ERRORS=1 make BOARD=esp8266-esp-12x -C tests/ssp/ flash test
```
is
```
START
main(): This is RIOT! (Version: 2020.04-devel-40-g11fee5-periph/gpio/ext_api_proposal_table_void)
calling stack corruption function
*** RIOT kernel panic:
ssp: stack smashing detected
```
So it seems that it works properly now with the new GCC version.

Reply:
By the way, I guess the problem with SSP on ESP8266 was not related to the old toolchain version. Rather, it was probably caused by the LTO. We had the same problem with ESP32, which I solved by changing `tests/ssp` with PR #12767.